sympy/assumptions/handlers/ntheory.py

```diff
@@ -246,3 +250,30 @@ def Basic(expr, assumptions):
                 return None
             return not _even
         return _integer
+
+
+AskCoprimeHandler = Dispatcher('AskCoprimeHandler')
+
+@AskCoprimeHandler.register(Expr, Expr)
+def Expr_Expr(x, y, *, assumptions):
+    if x == y:
+        return False
```

Comment:
1 and 1 are coprime.

Reply:
We should return False here though if either argument is not an integer, since we can't add that in the known facts.